### ISSUE TITLE
The latest version doesn't play video files from the local device storage

### ISSUE THREAD

**Issue by jmellicker** [jepiqueau/capacitor-video-player#89]
We are encountering an issue with playback of local video files in the 3.x.x version of the plugin (updated for Capacitor 3).

When specifying a remote (https) URL for the video file, the video loads and plays as expected.  But when we download the same `.mp4` video file to the local storage of the mobile device and try to load the video into `capacitor-video-player` using the local filepath, the video does not load, and there is no error message we can detect.

Is there a known incompatibility or issue with loading local video files in this latest update?

**NOTES:**

- The previous 2.x.x version of the plugin successfully loaded and played video files from the local device storage
- To verify it wasn’t a local filepath problem, we tried using the same local filepath to load the local file into `video.js`, and it played fine

**Comment by jepiqueau** [comment]
@jmellicker i assume that you are using an Android device. It seems that something change in Android 12 that i have not yet figure out.

**Comment by jmellicker** [comment]
@jepiqueau Local playback stopped working on iOS and Android devices at the same time (when we switched to Capacitor 3 and your latest code). Do you think it could be a change from Capacitor 2 to Capacitor 3?

**Comment by jepiqueau** [comment]
@jmellicker i do not think so. The app with capacitor 3 is running perfect in an Android API 21 .

**Comment by jepiqueau** [comment]
@jmellicker this is fixed in 3.7.4 , you need to add 
```
    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" android:maxSdkVersion="32"/>
```
in the manifest file of the app and you will be prompted for permissions the first time you want to access internal media file from Gallery or DCIM
